learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
```

Comment:
I would prefer to remove all these no coverage comments and just let the coverage score go down.  These will all likely get covered as soon as we have more code that exercises them anyway and it will be worth knowing that